Documentation/templates/coredata/swift3.md

```diff
@@ -0,0 +1,80 @@
+## Template Information
+
+| Name      | Description       |
+| --------- | ----------------- |
+| File name | coredata/swift3.stencil |
+| Configuration example | <pre>coredata:<br />  inputs: path/to/model.xcdatamodeld<br />  outputs:<br />    templateName: swift3<br />    output: CoreData.swift</pre> |
+| Language | Swift 3 |
+| Author | Grant J. Butler |
+
+## When to use it
+
+- When you need to generate *Swift 3* code.
+
+Note: the template will only generate code for entities that have set the `Codegen` property to "Manual/None". This is to avoid conflicts with Xcode, which will automatically generate code for the other options.
```

Comment:
👌

Reply:
Btw did you add some tests for this? (ie one xcdatamodel with things set toManual, another with Automatic)

Reply:
Yeah, check the context (here: https://github.com/grantjbutler/SwiftGen/blob/feature/core-data-parsing/Tests/Fixtures/StencilContexts/CoreData/defaults.yaml#L25). There are 2 entities `AutoClassGen` and `AutoExtensionGen` that have that option set to class/extension respectively.

The entities are still there, should other entities refer to these via relationships for example.

Reply:
Top notch 👌